src/System.Linq.Parallel/tests/WithCancellationTests.cs

```diff
@@ -185,17 +185,15 @@ public static void CTT_NonSorting_SynchronousMergerEnumeratorDispose()
             Assert.NotNull(caughtException);
         }
 
-        
-
         /// <summary>
-        /// 
+        ///
```

Comment:
Are all of the changes to this file just formatting, i.e. removing a space here and there?  I think you alluded to this separately, but we should probably keep all such trivial formatting changes as part of a separate cleanup done by a tool.